### PR TITLE
Main nav next story

### PR DESCRIPTION
**Pull request by rishabhnag1**
Finished main navigation, created GUID for each story to identify and push them onto stack, and finished Next Story navigation

### CODE REVIEW

File: src/UsfulPortfolio.js [hunk 1]
```diff
@@ -79,36 +67,24 @@ export default class UsfulPortfolio extends Component {
...
+    let globalNavigatorProps = {navigator};
     function getScene() {
       switch (route.id) {
-        case 'DetailedStoryDemoScene': {
-          return <DetailedStoryItem story={AppData.stories[0]} nextStory={AppData.stories[1]}/>;
-        }
-        case 'MainNavDemoScene': {
-          return <Portfolio/>;
+        case 'DetailedStoryScene': {
+          let storyId = AppData.stories.indexOf(AppData.stories.find(x => x.id === route.storyId));
```
Comment (clintonjrobinson) [inline src/UsfulPortfolio.js]: why not just do this?

```
let story = AppData.stories.find(x => x.id === route.storyId);

File: src/UsfulPortfolio.js [hunk 2]
```diff
@@ -79,36 +67,24 @@ export default class UsfulPortfolio extends Component {
...
+          console.log(storyId);
```
Comment (clintonjrobinson) [inline src/UsfulPortfolio.js]: Left a console.log here

File: src/helpers/formatters/guid.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+export default function guid() {
```
Comment (clintonjrobinson) [inline src/helpers/formatters/guid.js]: This can just be under helpers/guid.js since its not a formatter

File: src/components/Story/StoryCard.js [hunk 4]
```diff
@@ -59,22 +52,26 @@ let styles = StyleSheet.create({
...
 
   constructor(props) {
     super(props);
   }
 
 
-  openStories() {
-    return true;
+  openStories(e) {
+    console.log(this.props.id);
```
Comment (clintonjrobinson) [inline src/components/Story/StoryCard.js]: Left a console log here.

File: src/components/Story/StoryCard.js [hunk 5]
```diff
@@ -59,22 +52,26 @@ let styles = StyleSheet.create({
...
+    {this.props.navigator.push({
```
Comment (clintonjrobinson) [inline src/components/Story/StoryCard.js]: Don't need curlys around this.

```
this.props.navigator.push({
  id: 'DetailedStoryScene',
   storyId: this.props.id
});

File: src/UsfulPortfolio.js (hunk 1 above)
Comment (rishabhnag1) [inline src/UsfulPortfolio.js]: i wrote it this way to get the actual index of the story in the array, so I could increment the index to pass into the nextStory prop.
`nextStory={AppData.stories[storyId + 1]`
 is there a better way to do this that I'm blanking on?

Comment (clintonjrobinson) [inline src/UsfulPortfolio.js]:
```
let story = AppData.stories.findIndex(x => x.id === route.storyId);

Comment (clintonjrobinson) [inline src/UsfulPortfolio.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array

**Pull request merged**